contracts/SpokePool.sol

```diff
@@ -43,6 +43,16 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
     // Origin token to destination token routings can be turned on or off.
     mapping(address => mapping(uint256 => bool)) public enabledDepositRoutes;
 
+    // Associates relayer refund distribution proofs to unique IDs.
+    mapping(uint256 => bytes32) public relayerRefundRoots;
+
+    // Count of additions to `relayerRefundRoots`.
+    uint256 public relayerRefundRootCount;
+
+    // This is a 2D bitmap tracking which leafs in the relayer refund roots have been claimed, with max size of
+    // 256 elements, limiting us to 256 relayer refund leafs per distribution proof.
```

Comment:
this comment is wrong. we have no such limitation.

Reply:
fixed